src/GitTools.Core/GitTools.Core.csproj

```diff
@@ -19,36 +19,36 @@
     <DebugSymbols>true</DebugSymbols>
     <DebugType>full</DebugType>
     <Optimize>false</Optimize>
-    <OutputPath>..\..\output\debug\NET45\GitTools.Core\</OutputPath>
+    <OutputPath>..\..\output\Debug\GitTools.Core\net45\</OutputPath>
```

Comment:
GitTools.Core
GitTools.IssueTrackers
GitTools.Testing

I think we have more than 1 lib :-)